servicepulse/upgrades/supported-versions.md

```diff
@@ -0,0 +1,19 @@
+---
+title: ServicePulse Supported Versions
+summary: Supported versions of ServicePulse
+reviewed: 2022-02-28
```

Comment:
How was this reviewed 6 months ago if it's brand new? 😆 But I'd inline it into the support policy page anyway.